sig-list.md

```diff
@@ -49,7 +49,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Release](sig-release/README.md)|[release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease)|* [Stephen Augustus](https://github.com/justaugustus), Cisco<br>* [Sascha Grunert](https://github.com/saschagrunert), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-release)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-release)|* Regular SIG Meeting APAC/EU Oriented Time: [Tuesdays at 15:30 UTC (biweekly)](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09)<br>* Regular SIG Meeting US Oriented Time: [Tuesdays at 17:00 UTC (biweekly)](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09)<br>
 |[Scalability](sig-scalability/README.md)|[scalability](https://github.com/kubernetes/kubernetes/labels/sig%2Fscalability)|* [Marcel Zieba](https://github.com/marseel), Google<br>* [Shyam Jeedigunta](https://github.com/shyamjvs), AWS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scalability)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scale)|* Regular SIG Meeting: [Thursdays at 10:30 PT (Pacific Time) (bi-weekly ([upcoming meeting dates](#upcoming-meeting-dates)))](https://zoom.us/j/94252896018?pwd=cTlMMlBoTHZqUEdjRm9VY2NWNUg5dz09)<br>
-|[Scheduling](sig-scheduling/README.md)|[scheduling](https://github.com/kubernetes/kubernetes/labels/sig%2Fscheduling)|* [Wei Huang](https://github.com/Huang-Wei), Apple<br>* [Abdullah Gharaibeh](https://github.com/ahg-g), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scheduling)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scheduling)|* biweekly meeting (NA & Europe): [Thursdays at 17:00 UTC (biweekly starting Thursday June 7, 2018)](https://zoom.us/j/841218129)<br>* monthly meeting (APAC): [first Thursdays at 02:00 UTC (monthly)](https://zoom.us/j/841218129)<br>
+|[Scheduling](sig-scheduling/README.md)|[scheduling](https://github.com/kubernetes/kubernetes/labels/sig%2Fscheduling)|* [Wei Huang](https://github.com/Huang-Wei), Apple<br>* [Abdullah Gharaibeh](https://github.com/ahg-g), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scheduling)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scheduling)|* biweekly meeting (NA & Europe): [Thursdays at 17:00 UTC (biweekly starting Thursday June 7, 2018)](https://zoom.us/j/841218129)<br>* descheduler biweekly meeting: [Tuesdays at 17:00 UTC (biweekly starting Tuesday September 13, 2022)](https://meet.google.com/iri-uqrk-wdm)<br>* monthly meeting (APAC): [first Thursdays at 02:00 UTC (monthly)](https://zoom.us/j/841218129)<br>
```

Comment:
17:00 UTC， this time i am sorry that i can not join, since i locate on UTC+8 :(

Reply:
I'm sorry @JaneLiuL  :\ we could use the current sig-scheduling APAC meeting to go over anything of note from the descheduler calls, would that work for you?

Otherwise I think it will be tough to find a good time for one meeting with that much of a distribution

Reply:
Do we have more people active in the descheduler project that are close to that timezone? @JaneLiuL another option is for someone to host another dedicated meeting in a convenient time for those in that area. @JaneLiuL would you like to host it?